.ci/linux-build.sh

```diff
@@ -28,20 +28,14 @@ function configure_ovn()
 save_OPTS="${OPTS} $*"
 OPTS="${EXTRA_OPTS} ${save_OPTS}"
 
-# If AddressSanitizer or UdefinedBehaviorSanitizer is requested, enable it,
+# If AddressSanitizer and UdefinedBehaviorSanitizer are requested, enable them,
```

Comment:
`UdefinedBehaviorSanitizer`

should be 

`UnderfinedBehaviorSanitizer`

but I will not restart the tests right now because they still fail :(